### PR TITLE
Add BVN networking models

### PR DESCRIPTION
**Pull request by vanshg**
Story: https://app.shortcut.com/smileid/story/9097/add-bvn-enforced-consent-screens-to-the-v10-sdk-ios

## Summary

Starts the work for adding BVN Consent. This adds the API calls and models

### CODE REVIEW

File: Example/SmileID.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -670,15 +670,15 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
-				CURRENT_PROJECT_VERSION = 23;
+				CURRENT_PROJECT_VERSION = 26;
```
Comment (jumaallan) [inline Example/SmileID.xcodeproj/project.pbxproj:673]: Does this file need to be committed?

File: Example/SmileID.xcodeproj/xcshareddata/xcschemes/SmileID-Example.xcscheme [hunk 2]
```diff
@@ -65,8 +65,8 @@
    </TestAction>
    <LaunchAction
       buildConfiguration = "Debug"
-      selectedDebuggerIdentifier = ""
-      selectedLauncherIdentifier = "Xcode.IDEFoundation.Launcher.PosixSpawn"
+      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
```
Comment (jumaallan) [inline Example/SmileID.xcodeproj/xcshareddata/xcschemes/SmileID-Example.xcscheme:68]: and this one?

File: Example/SmileID.xcodeproj/project.pbxproj (hunk 1 above)
Comment (vanshg) [reply Example/SmileID.xcodeproj/project.pbxproj:673]: Yep. Xcode maintains a lot of generated files that do need to be committed

File: Example/SmileID.xcodeproj/xcshareddata/xcschemes/SmileID-Example.xcscheme (hunk 2 above)
Comment (vanshg) [reply Example/SmileID.xcodeproj/xcshareddata/xcschemes/SmileID-Example.xcscheme:68]: Ditto

File: Example/SmileID.xcodeproj/project.pbxproj (hunk 1 above)
Comment (vanshg) [reply Example/SmileID.xcodeproj/project.pbxproj:673]: This change here is bumping the build number for the updated TestFlight release

**Review by jumaallan — APPROVED** [review]

File: Example/SmileID.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jumaallan) [reply Example/SmileID.xcodeproj/project.pbxproj:673]: Aah

Thanks

**Pull request merged**